Intersect.Server/Entities/Events/Conditions.cs

```diff
@@ -436,18 +436,14 @@ QuestBase questBase
             QuestBase questBase
         )
         {
-            for (var i = 0; i < Options.EquipmentSlots.Count; i++)
+            if (player == null || condition == null)
```

Comment:
Why is QuestBase even a parameter? I don't see it used in this method?

Same thing for EventInstance?